wgpu/utils/shadertoy.py

```diff
@@ -1,9 +1,14 @@
 import time
 import ctypes
+import numpy as np
+from PIL import Image
```

Comment:
This gets a bit tricky in terms of dependencies. I'd prefer to make `snapshot()` return a memoryview object, that can be easily converted to a numpy array (can mention in the docs that ppl can use `np.asarray()`.

Reply:
definitely, I haven't even looked up what dependencies are included, and looks like numpy is just in the dev requirements. 
Will shorten the method and the copy over the docstring